src/App.jsx

```diff
@@ -1,5 +1,11 @@
+import { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { GoodList } from './components/goodList';
```

Comment:
Ensure that the import path for `GoodList` is correct and matches the actual file structure. Also, verify the case sensitivity of the file name as it might cause issues in some environments.